source/develop/release-management/features/storage/incremental-backup.md

```diff
@@ -266,6 +267,12 @@ You need to poll the backup until the phase is `"ready"`.
 Once the backup is ready the response will include `<to_checkpoint_id>` which
 should be used as the `<from_checkpoint_id>` in the next incremental backup.
 
```

Comment:
We should describe the normal flow first, when all disks have backup_mode="incremental".

Reply:
The normal flow starts with full and then incremental, we cannot jump to incremental.
 Also, I've changed a little bit the explanation order so it should  fine now.